examples/static_files/src/tests.rs

```diff
@@ -0,0 +1,71 @@
+use std::fs::File;
+use std::io::Read;
+use rocket::testing::MockRequest;
+use rocket::http::Method::*;
+use rocket::http::Status;
+use rocket::response::Body;
+
+use super::rocket;
+
+fn body_to_bytes(body: Body<&mut Read>) -> Option<Vec<u8>> {
+    match body {
+        Body::Sized(b, _)  => {
```

Comment:
Just as a note, you can match on both variants at the same time to avoid the duplication like this:

```rust
match body {
    Body::Sized(b, _) | Body::Chunked(b, _) => {
        ...
    }
}
```